rqt_joint_trajectory_controller/src/rqt_joint_trajectory_controller/joint_limits_urdf.py

```diff
@@ -33,12 +33,15 @@ def get_joint_limits(key='robot_description', use_smallest_joint_limits=True):
             if jtype == 'fixed':
                 continue
             name = child.getAttribute('name')
+            try:
+                limit = child.getElementsByTagName('limit')[0]
+            except:
+                continue
```

Comment:
I admit that it might strike as odd to a user if a controller is running, but does not show up in the combo box.

Ideally, one would have to keep a list of all controllers that are running, but don't have a valid limits spec (hence don't show up in the combo), and print a warning *once* every time one of these is found. I say once, because the plugin is [periodically checking for running controllers](https://github.com/ros-controls/ros_controllers/blob/rqt-jtc-improvements/rqt_joint_trajectory_controller/src/rqt_joint_trajectory_controller/joint_trajectory_controller.py#L174-L177), so the combos are in sync with the system.

It's doable, just more work.